docs/print.rst

```diff
@@ -6,6 +6,7 @@ The header ``<kitty/print.hpp>`` implements functions to print truth tables.
 .. doc_brief_table::
    print_binary
    print_hex
+   print_k_map
```

Comment:
The documentation is generated by matching the function name and fetching the comment written before the function. Your function name is `print_kmap`.

Reply:
Thank you, I have just fixed it!